src/class/domain/classArticle/classArticle.domain.reader.ts

```
@@ -0,0 +1 @@
+export interface ClassArticleDomainReader {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)